src/Java.Interop.Tools.TypeNameMappings/Java.Interop.Tools.TypeNameMappings/JavaNativeTypeManager.cs

```diff
@@ -143,7 +151,7 @@ public static string ToJniName (Type type)
 				"java/lang/Object";
 		}
 
-		public static string ToJniName (Type type, ExportParameterKind exportKind)
+		static string ToJniName (Type type, ExportParameterKind exportKind)
```

Comment:
`Mono.Android.dll` requires this method in [`JNIEnv.GetJniName(Type)`](https://github.com/xamarin/xamarin-android/blob/5951869/src/Mono.Android/Android.Runtime/JNIEnv.cs#L885). It can't be made `private`, at least not yet?

Reply:
This is what JNIEnv.GetJniName() does:
https://github.com/xamarin/xamarin-android/blob/0d16153f0239602fa96dd5876b1e400c2dd2cdea/src/Mono.Android/Android.Runtime/JNIEnv.cs#L885

It only references GetJniName(Type), not GetJniName(Type,ExportParameterKind).